plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3369,4 +3368,171 @@ public void testTargetMaxFileSize()
                 // so just to be safe we check if it is not much bigger
                 .forEach(row -> assertThat((Long) row.getField(0)).isBetween(1L, maxSize.toBytes() * 3));
     }
+
+    @Test
+    public void testVacuumParameterValidation()
+    {
```

Comment:
add test with retention passed as arg shorted than `vacuum_min_retention`

Reply:
Added a case to `testVacuumParameterValidation`